types/react-router-redux/index.d.ts

```diff
@@ -49,30 +52,4 @@ export const routerActions: {
     goForward: typeof goForward
 };
 
-export interface LocationActionPayload {
```

Comment:
Why do you want to remove the typings for these interfaces? They are not redundant.